src/replace_landsurface_with_FF_IC.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 ACCESS-NRI (https://www.access-nri.org.au/)
+# See the top-level COPYRIGHT.txt file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Created by: Chermelle Engel <Chermelle.Engel@anu.edu.au>
+
+import iris
+import mule
+import os
+import sys
+import numpy as np
+from glob import glob
+from pathlib import Path
+import xarray as xr, sys, argparse
+from datetime import datetime,timedelta
```

Comment:
```suggestion
import mule
```
Just suggesting the removal of the unused imports here :)